### PR TITLE
feat: landing

### PR DESCRIPTION
**Pull request by BalogunofAfrica**
Add Colighter Landing page based on Tailwind template

### CODE REVIEW

File: apps/web/src/ComingSoon.tsx [hunk 1]
```diff
@@ -1,18 +1,953 @@
-import React from "react";
```
Comment (okjodom) [inline apps/web/src/ComingSoon.tsx:1]: we should delete this page and fully replace with the landing page

File: apps/web/src/index.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+@tailwind base;
```
Comment (okjodom) [inline apps/web/src/index.css:1]: any chance we can keep all css within the tsx?

File: apps/web/src/ComingSoon.tsx (hunk 1 above)
Comment (BalogunofAfrica) [reply apps/web/src/ComingSoon.tsx:1]: Alrighty

**Comment by BalogunofAfrica** [comment]
@okjodom have any copy in mind?

File: apps/web/src/index.css (hunk 2 above)
Comment (okjodom) [reply apps/web/src/index.css:1]: yep. not blocking, just a code pattern recommendation to try achieve in future

**Pull request merged**